BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -3443,6 +3443,14 @@ public float BallisticTrajectorySimulation(ref Vector3 position, Vector3 velocit
                         ++i;
                     }
             }
+            if (!BDArmorySettings.DEBUG_LINES)
```

Comment:
This isn't necessary. The trajectory renderer gets disabled at the start of `OnGUI` if `DEBUG_LINES` is disabled and there isn't really any reason to set it to `null` since it'll get destroyed with the `ModuleWeapon`.